templates/advertisementtiles.mustache

```diff
@@ -0,0 +1,77 @@
+{{!
+    This file is part of Moodle - http://moodle.org/
+
+    Moodle is free software: you can redistribute it and/or modify
+    it under the terms of the GNU General Public License as published by
+    the Free Software Foundation, either version 3 of the License, or
+    (at your option) any later version.
+
+    Moodle is distributed in the hope that it will be useful,
+    but WITHOUT ANY WARRANTY; without even the implied warranty of
+    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+    GNU General Public License for more details.
+
+    You should have received a copy of the GNU General Public License
+    along with Moodle.  If not, see <http://www.gnu.org/licenses/>.
+}}
+{{!
+    @template theme_boost_union/infobanners
+
+    Boost Union Info banners layout template.
+
+    Context variables required for this template:
+    * advtiles - The configured info banners list with:
+      * content - The info banner content
+      * backgroundimg = Image to be displayed behind the content;
+      * linktitle = The link title which is displayed (might be empty);
+      * link = The actual URL (might be empty);
+      * order = Sequence to display tiles. (the array is orders according to this number);
+      * no = Order which is displayed in the settings.;
+    Example context (json):
+    {
+        "advtilescolumns" : 4;
+        "advtiles": [
+            {
+                "content": "Das hier ist eine tolle Moodle Plattform",
+                "backgroundimgurl": "",
+                "linktitle": "",
+                "link": "",
+                "no": 2,
+                "order": 1
+            },
+            {
+                "content": "Sie haben Hunger?",
+                "backgroundimgurl": "",
+                "linktitle": "Mensa",
+                "link": "https://stw-muenster.de/",
+                "no": 1,
+                "order": 3
+            }
+        ]
+    }
+    -- When changing the divs behat xpath might fail!
+}}
+<div class="container">
+    <div class="row themeboostunionadvtileswrapper" >
+        {{#advtiles}}
+        <div id="themeboostunionadvtiles{{no}}" class="col {{advtileslayoutclass}} align-items-stretch d-flex align-self-stretch p-2">
```

Comment:
The align-* classes are fine and correct to align the cards vertically, but the classes do nothing to wrap the grid on smaller screens yet.

We have not only to add col-* classes but only classes like col-md-* to leverage Bootstrap's responsive grid. This is best done in
/layout/includes/advertisementtiles.php.